app/views/tournaments/new.html.erb

```diff
@@ -1,2 +1,17 @@
 <h1>Tournaments#new</h1>
 <p>Find me in app/views/tournaments/new.html.erb</p>
+<%= form_for(@tournament) do |f| %>
+  <%= f.label :name %>
+  <%= f.text_field :name %>
+  <%= f.label :description %>
+  <%= f.text_field :description %>
+  <%= f.label :location %>
+  <%= f.text_field :location %>
+  <%= f.label :number_of_teams %>
+  <%= f.number_field :number_of_teams %>
+  <%= f.label :number_of_players_per_team %>
+  <%= f.number_field :number_of_players_per_team %>
+  <%= f.label :price %>
+  <%= f.number_field :price %>
+  <%= f.submit %>
+<% end %>
```

Comment:
Also here, **use simple form!!!**

Instead of repeating yourself please be **DRY**, create a partial here `app/views/restaurants/_tournament_form.html.erb` and put the code of the form

```
<%= simple_form_for(@tournament) do |f| %>
  <%= f.input :name %>
  <%= f.input :description %>
  <%= f.input :location %>
  <%= f.input :number_of_teams %>
  <%= f.input :number_of_players_per_team %>
  <%= f.input :price %>
  <%= f.submit %>
<% end %>
```

```suggestion
<%= render "tournament_form", tournament: @tournament %>
```